src/test/regress/expected/multi_insert_select.out

```diff
@@ -1126,7 +1129,8 @@ FROM   (SELECT SUM(raw_events_second.value_4) AS v4,
                raw_events_second
         WHERE  raw_events_first.user_id = raw_events_second.user_id
         GROUP  BY raw_events_second.value_3) AS foo;
-DEBUG:  Group by list without distribution column is not allowed  in distributed INSERT ... SELECT queries
+DEBUG:  cannot push down this subquery
+DETAIL:  Group by list without partition column is currently unsupported when a subquery references a column from another query
```

Comment:
DeferErrorIfUnsupportedSubqueryPushdown assumes that the input query is a subquery, hence the debug message is misleading